src/main/steps/applicant2/address/change/content.ts

```diff
@@ -0,0 +1,60 @@
+import { YesOrNo } from '../../../../app/case/definition';
```

Comment:
Similar pages used twice can be made common

Reply:
okay will try this, should resolve my sonar issues